newIDE/app/src/UI/RaisedButtonWithSplitMenu.js

```diff
@@ -11,7 +11,7 @@ import { Spacer } from './Grid';
 type Props = {|
   id?: string,
   label?: React.Node,
-  primary?: boolean,
+  primary: true, // Force making only primary raised split buttons.
```

Comment:
maybe make the prop non mandatory, so we don't think about it.
And yet, we have a warning if we try to set false?

Reply:
There are a few places in the code where this button is used, without setting this prop, you may have to change those

Reply:
In theory Flow should error if there is any prop where primary is not set to true. I should have replaced these by FlatButtonWithSplitMenu - so that it's clear.
I'll keep this so that anyone trying to do a non primary RaisedButtonWithSplitMenu think twice about really doing it.